tests/integration/targets/vmware_esxi_maintenance_mode/tasks/eco-vcenter.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Import common vars
+  ansible.builtin.include_vars:
+    file: ../group_vars.yml
+
+- name: Test
+  block:
+    - name: Deploy Virtual ESXi Host
+      community.vmware.vmware_content_deploy_ovf_template:
```

Comment:
Maybe better to use the  `vmware.vmware_rest.vcenter_ovf_libraryitem` module to deploy a VM from OVF template?
And also add a resource pool please, for better visibility.